### PR TITLE
Update members and roles

### PR DESCRIPTION
**Pull request by agnauck**
Update members and roles as per Q3-2020 and board&council elections

### CODE REVIEW

File: data/clients.json [hunk 1]
```diff
@@ -636,7 +636,7 @@
         "last_renewed": "2020-12-10T19:00:55",
         "name": "txmmp",
         "platforms": [
-            "Unix"
+            "Linux"
```
Comment (Zash) [inline data/clients.json:639]: This looks like something that should not be part of this PR

Comment (agnauck) [reply data/clients.json:639]: I only added this because the build validation was always failing with linter errors

Comment (agnauck) [reply data/clients.json:639]: But I can remove when you want to fix the linter errors in a separate PR
